Gems/Terrain/Code/Source/EditorComponents/EditorTerrainPhysicsColliderComponent.h

```diff
@@ -9,13 +9,26 @@
 #pragma once
 
 #include <Components/TerrainPhysicsColliderComponent.h>
-#include <AzToolsFramework/ToolsComponents/EditorComponentBase.h>
 #include <LmbrCentral/Component/EditorWrappedComponentBase.h>
+#include <AzFramework/Terrain/TerrainDataRequestBus.h>
+#include <AzToolsFramework/API/ToolsApplicationAPI.h>
+#include <AzToolsFramework/ToolsComponents/EditorComponentBase.h>
+#include <AzToolsFramework/Viewport/ViewportMessages.h>
 
 namespace Terrain
 {
+    struct DebugImageQuad
```

Comment:
If this struct is only meant to be internal to be used by `EditorTerrainPhysicsColliderComponent::m_debugQuads;`, then please move the definition of the struct inside `EditorTerrainPhysicsColliderComponent` too, as it's no use to those including this header.